artifacts/overlays/ingress-mode/.gitignore

```diff
@@ -0,0 +1 @@
+.dockerconfigjson
```

Comment:
no end of line

Reply:
I merge the inner .gitignore to the outer one